package.json

```diff
@@ -4,9 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit",
+    "test": "npm run lint && npm run unit && npm run cover && npm run snyk",
     "unit": "mocha --recursive test/spec/ --require test/helper --timeout 300000",
-    "lint": "eslint ./lib/ ./index.js"
+    "cover": "istanbul cover _mocha -- --recursive test/spec/ --require test/helper --timeout 300000",
+    "check-coverage": "istanbul check-coverage --statement 100 --branch 100 --function 100 --line 100",
```

Comment:
I'm guessing we don't actually have this much coverage which is why you don't have check-coverage in the script for test.

If that's the case what do people think about setting the limits to just below what we currently have, that way we can at least ensure that we don't go backwards and can slowly 'ratchet' the limits up over time?